src/components/capture/creation/index.tsx

```diff
@@ -107,8 +110,8 @@ function NewCaptureModal() {
     //const [availableSchemas, setAvailableSchemas] = useState<any[]>([]);
```

Comment:
Can the schema editor to-dos be removed? And would you mind elaborating on the approach you intended to take here?

Reply:
No - because these are not for the _Schema_ Editor they are for an _Editor_ of Schemas. Meaning allowing the customer to choose what schemas that want to bring in from a connector.